spec/factories/supervisor_reports.rb

```diff
@@ -5,10 +5,7 @@
     pictures_saved { [true, false].sample }
     passenger_statement { FFaker::Lorem.paragraphs(5).join "\n" }
     faxed { [nil, Time.zone.now].sample }
-    completed_drug_or_alcohol_test { [true, true, false].sample }
-    completed_drug_test { [true, false].sample }
-    completed_alcohol_test { [true, false].sample }
-    reason_test_completed { SupervisorReport::REASONS_FOR_TEST.sample }
+    test_status { SupervisorReport::REASONS_FOR_TEST.sample }
```

Comment:
Instead of using `sample` which leads to variable test conditions / results (as we've seen), let's just give it one status.